### PR TITLE
Added new file for line number access

### PR DESCRIPTION
**Pull request by rpetrusha**
## Added new file for line number access

Related to dotnet/docs#13958

### CODE REVIEW

File: snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-﻿App.xaml.vb: ~/docs/framework/wpf/app-development/application-management-overview.md
+﻿Application.xaml.vb: ~/docs/framework/wpf/app-development/application-management-overview.md
```
Comment (mairaw) [inline snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt]:
```suggestion
application.xaml.vb: ~/docs/framework/wpf/app-development/application-management-overview.md
```

File: snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt [hunk 2]
```diff
@@ -1 +1,2 @@
...
+Application.xaml.vb: ~/docs/framework/wpf/app-development/persist-and-restore-application-scope-properties.md
```
Comment (mairaw) [inline snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt]:
```suggestion
application.xaml.vb: ~/docs/framework/wpf/app-development/persist-and-restore-application-scope-properties.md
```

File: snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt (hunk 1 above)
Comment (mairaw) [reply snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt]: should we perhaps link to the file on GitHub for easier access?

```suggestion
application.xaml.vb: https://github.com/dotnet/docs/blob/master/docs/framework/wpf/app-development/application-management-overview.md
```

**Comment by tdykstra** [comment]
@mairaw @rpetrusha How is this file used? Is it an automated system to warn when source code changes affect line numbers or are people supposed to manually look for these files when changing source code?  Is the process documented somewhere, and are there alternatives / has this process been assessed to determine if it's worth the overhead it adds?

**Comment by Youssef1313** [comment]
Maybe missing parallel changes at c# ?
https://github.com/dotnet/samples/blob/master/snippets/csharp/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/CSharp/access-by-line.txt

File: snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt (hunk 1 above)
Comment (rpetrusha) [reply snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt]: The link is a good idea, @mairaw. I'll modify the other files in a separate PR to include links.

**Comment by rpetrusha** [comment]
The details are in dotnet/docs#10241, @tdykstra. It isn't automated -- it hopefully alerts us that when a change is made to one of the files that's referenced by line number, we can manually update the line number references. So far, AFAIK, no line number references have been updated to assess how it works.

**Pull request merged**